openssl-dynamic/src/main/c/sslcontext.c

```diff
@@ -100,6 +100,17 @@ TCN_IMPLEMENT_CALL(jlong, SSLContext, make)(TCN_STDARGS, jint protocol, jint mod
 
     UNREFERENCED(o);
 
+#if OPENSSL_VERSION_NUMBER >= 0x10100000L && !defined(LIBRESSL_VERSION_NUMBER)
+    // TODO this is very hacky as io.netty.handler.ssl.OpenSsl#doesSupportProtocol also uses this method to test for supported protocols. Furthermore
```

Comment:
Thanks for clarifying. As an intermediate step I don't think we should prevent using the deprecated methods until we find a viable replacement. See https://github.com/netty/netty-tcnative/issues/257.